.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
please remove this change,
this could be irrelevant for others,

you could add this to the .gitignore file instead of deleting it on your system.
Then pushing it should do the job.

Reply:
```
.vscode/
```
Add this to .gitignore and then push.
this change should do the job.